scrapers/StateOfTexas.mjs

```diff
@@ -1,32 +1,53 @@
 import fetch from 'node-fetch';
 import moment from 'moment';
 
+const baseUrl = 'https://api.tnris.org/api/v1';
+let collectionIds = null;
```

Comment:
This cache isn't a great idea, it's over complex and is confusing. The scrapers aren't run constantly, only on a infrequent but still reasonable schedule.